README.md

```diff
@@ -78,7 +78,7 @@ MCore consists of two parts:
 
 ## MExpr
 
-One design objective of MExpr is to make the concrete syntax very close to the abstract syntax of the language. That is, no syntactic sugar is introduced to the concrete MCore syntax. The MExpr language is not intended to be a general purpose programming language, but a core language to which other languages translate into.
+One design objective of MExpr is to make the concrete syntax very close to the abstract syntax of the language. That is, no syntactic sugar is introduced to the concrete MCore syntax. The MExpr language is not intended to be a general purpose programming language, but as a core language to which other languages translate into.
```

Comment:
"but a core language" sounds better than "but as a core language"

Reply:
Well, I disagree. Anyway, I have now reformulated the sentence by breaking it up into two sentences instead.